### PR TITLE
Added copyright action & texts in all files

### PR DESCRIPTION
**Pull request by tusharojha**
The actual script to compare the copyright text in a file is extracted to another package at [tusharojha/copyright-check](https://github.com/tusharojha/copyright-check). 

We can:
- move the github repo for the action to dappforce 
- OR, I release a version on github marketplace and use it (might take some time to review 1-2 day)

### CODE REVIEW

File: copyright.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0.
+// Copyright (C) 2021-2022 DAPPFORCE PTE. Ltd., aleksandr.siman@gmail.com.
```
Comment (olehmell) [inline copyright.txt:2]:
```suggestion
// Copyright (C) 2021-2023 DAPPFORCE PTE. Ltd., aleksandr.siman@gmail.com.
```

Comment (tusharojha) [reply copyright.txt]: @olehmell done

**Comment by olehmell** [comment]
I will move it to our dappforce repo today

File: copyright.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0.
+// Copyright (C) 2021-2023 DAPPFORCE PTE. Ltd., dappforce@gmail.com.
```
Comment (olehmell) [inline copyright.txt:2]:
```suggestion
// Copyright (C) DAPPFORCE PTE. LTD., dappforce@gmail.com.
// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0.
```

**Review by olehmell — CHANGES_REQUESTED** [review]

**Comment by F3Joule** [comment]
Replaced with #239

**Pull request closed without merging**